ts/routes/deck-options/SpinBoxFloatRow.svelte

```diff
@@ -23,9 +23,12 @@
         <slot />
     </Col>
     <Col --col-size={6} breakpoint="xs">
-        <ConfigInput>
-            <SpinBox bind:value {min} {max} {step} {percentage} bind:focused />
-            <RevertButton slot="revert" bind:value {defaultValue} />
-        </ConfigInput>
+        <Row class="flex-grow-1">
```

Comment:
Not something that needs to be done in this PR, but just a note that the SpinBoxFloatRow and SpinBoxRow components are almost identical and can probably be refactored into a single component.